### DISCUSSION TITLE
I'm stuck at this

### DISCUSSION THREAD

**Discussion by AwaisCoder** [TheAlgorithms/Python#11181] [Q&A]
I'm getting this error while reading data from a CSV file. I'm creating a Diabetes prediction system using Django on PyCharm

![IMG-20231126-WA0004.jpg](https://github.com/TheAlgorithms/Python/assets/121681506/636b0288-7f12-47e6-ab4d-e9d364867e65)

I have checked the casing and the spacing there seems to be nothing wrong with that. Idk what to do but please help me if you can. I will provide with all the info needed to solve this

**Comment by yashbishnoi** [comment]
Can you share a code snippet where you are you using pandas

**Comment by eminsk** [comment]
The error message indicates that there is a KeyError for the key 'Outcome' in the axis. This might mean that the 'Outcome' column is not present in the dataframe that you are trying to drop.
To troubleshoot this issue, you can try the following steps:
1. Check if the 'Outcome' column exists in your dataframe. You can do this by printing the column names of your dataframe and verifying if 'Outcome' is present in the list.
2. If the 'Outcome' column is not present in your dataframe, you might want to check if the column name is misspelled or if there are any leading or trailing spaces in the column name.
3. If the 'Outcome' column is present in your dataframe, you might want to check if you are using the correct dataframe to drop the column. It is possible that you are using a different dataframe than the one you think.
4. If all 3 of my points did not help, then for a more accurate answer you only need to present your code

**Comment by tianyizheng02** [comment]
The discussion section of this repo is not for general Python help. Posts should be about the repository and its contents.
